File: server/trs/trs/MODEL/car.cs
using System; 
namespace TakeRing.Model
{
	public class car
	{
   		     
      	/// <summary>
		/// car表主键
        /// </summary>
        public long id { get; set; }
		/// <summary>
		/// 用户id
        /// </summary>
        public long user_id { get; set; }
		/// <summary>
		/// 出发日期
        /// </summary>
        public DateTime startDate { get; set; }
		/// <summary>
		/// 出发时间
        /// </summary>
        public TimeSpan startTime { get; set; }
		/// <summary>
		/// 联系方式
        /// </summary>
        public string phone { get; set; }
		/// <summary>
		/// 车牌号
        /// </summary>
        public string car_number { get; set; }
		/// <summary>
		/// 车的名字
        /// </summary>
        public string car_name { get; set; }
		/// <summary>
		/// 空位数
        /// </summary>
        public int car_sum { get; set; }
		/// <summary>
		/// 目的地
        /// </summary>
        public string car_destination { get; set; }
		/// <summary>
		/// 出发地
        /// </summary>
        public string car_origin { get; set; }
		/// <summary>
		/// 类别：0为顺风车，1为空位
        /// </summary>
        public int type { get; set; }
		   
	}
}

File: server/trs/trs-service-client/src/main/java/cn/autumn/trs/client/basic/car.java
package cn.autumn.trs.client.basic;

public class car {
    private Long id;

    private Long userId;

    private String startdate;

    private String starttime;

    private String phone;

    private String carNumber;

    private String carName;

    private Integer carSum;

    private String carDestination;

    private String carOrigin;

    private Integer type;

    public Long getId() {
        return id;
    }

    public void setId(Long id) {
        this.id = id;
    }

    public Long getUserId() {
        return userId;
    }

    public void setUserId(Long userId) {
        this.userId = userId;
    }

    public String getStartdate() {
        return startdate;
    }

    public void setStartdate(String startdate) {
        this.startdate = startdate == null ? null : startdate.trim();
    }

    public String getStarttime() {
        return starttime;
    }

    public void setStarttime(String starttime) {
        this.starttime = starttime == null ? null : starttime.trim();
    }

    public String getPhone() {
        return phone;
    }

    public void setPhone(String phone) {
        this.phone = phone == null ? null : phone.trim();
    }

    public String getCarNumber() {
        return carNumber;
    }

    public void setCarNumber(String carNumber) {
        this.carNumber = carNumber == null ? null : carNumber.trim();
    }

    public String getCarName() {
        return carName;
    }

    public void setCarName(String carName) {
        this.carName = carName == null ? null : carName.trim();
    }

    public Integer getCarSum() {
        return carSum;
    }

    public void setCarSum(Integer carSum) {
        this.carSum = carSum;
    }

    public String getCarDestination() {
        return carDestination;
    }

    public void setCarDestination(String carDestination) {
        this.carDestination = carDestination == null ? null : carDestination.trim();
    }

    public String getCarOrigin() {
        return carOrigin;
    }

    public void setCarOrigin(String carOrigin) {
        this.carOrigin = carOrigin == null ? null : carOrigin.trim();
    }

    public Integer getType() {
        return type;
    }

    public void setType(Integer type) {
        this.type = type;
    }
}